### PR TITLE
Improve MD to HTML

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard5**
## Summary

This is a refactor for the Markdown parsing that happens on build-time. 

This PR
- replaces `snarkdown` and `dompurify` with `unified` + `remark-parse`,  
- adds support for `externalLinks` to Markdown in `MDToHTMLString`
- removes `{{whitespace}}` from Lokalise text and use original Markdown newlines (`\n`)
- removes `replaceLinks/index.tsx` since it is no longer needed
- applies `MDToHTMLString` to `over` page to load MD to props using `getStaticProps`

## Motivation

Currently newlines in the Markdown from Lokalise are brought into the content through a function that replaces variables, like this:  `replaceVariablesInText(str, {
    whitespace: '\n\n \n\n',
  })`. Unified's parsers detect newlines better and allows us to add more modules into one ecosystem to perform more actions on the MD. Now the newlines work more naturally from Lokalise, and we have control over how external links are treated in the copy. 

## Detailed design

- Add a selection of modules from the [UnifiedJS ecosystem](https://unifiedjs.com/)
- Remove the `{{whitespace}}` variables from text on Lokalise and perform other small changes to the `nl.json` and `en.json` files. This was done through a build and then I merged into the Lokalise branch from this feature branch.
- Use `getStaticProps` to pass the MD copy to props on build 
- Use `Fragment` to wrap the `<dt>` and `<dd>` used for the copy in order to assign it a `key`
- Setup the config for the `unified` processor in `utils/MDToHTMLString.ts`, using `remark-parse` to parse the Markdown, `remark-external-links` as a plugin to handle the links, `remark-rehype` to bridge to `rehype` and lastly `rehype-stringify` to make up the HTML string.

### CODE REVIEW

File: src/locale/en.json [hunk 1]
```diff
@@ -331,7 +331,7 @@
       },
       {
         "vraag": "I’ve found a technical vulnerability in the website. Where can I report this?",
-        "antwoord": "We work together with the National Cyber Security Centre (NCSC). Please notify them of the vulnerability as soon as possible using their form (https://english.ncsc.nl/contact/reporting-a-vulnerability-cvd) before making the issue publicly known. We will deal with your notification and resolve it as quickly as possible together with the NCSC."
+        "antwoord": "Wij werken samen met het Nationaal Cyber Security Centrum (NCSC). Stel het NCSC zo snel mogelijk op de hoogte via [het formulier](https://www.ncsc.nl/contact/kwetsbaarheid-melden/cvd-meldingen-formulier) voordat u dit aan de buitenwereld kenbaar maakt. In samenwerking met het NCSC nemen we de bevinding in behandeling en lossen dit zo snel mogelijk op."
```
Comment (VWSCoronaDashboard4) [inline src/locale/en.json:334]: I guess the translations should not be reverted.

File: src/pages/over.tsx [hunk 2]
```diff
@@ -23,7 +25,35 @@ interface IVraagEnAntwoord {
...
+
+// We use lokalise.com as our dictionary/text source and to support internationalisation.
+// Lokakise will output JSON files which can be found in `src/locale`.
+// However, all content lives inside plain strings. To support structured content and newlines,
+// we (optionally) write markdown in Lokakise and parse it to HTML.
+//
+// Ideally this entire page would have been build from markdown, but that’s not possible
+// with our internationalisation setup.
+export async function getStaticProps(): Promise<StaticProps> {
+  const text = require('../locale/index').default;
```
Comment (VWSCoronaDashboard4) [inline src/pages/over.tsx:42]: I see this is copied from the verantwoording.tsx; perhaps import it at the top of the file like is done in most places: ```import siteText from 'locale';```

File: src/utils/MDToHTMLString.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
-// @ts-ignore
-import snarkdown from 'snarkdown';
-// @ts-ignore
-import createDOMPurify from 'dompurify';
-// @ts-ignore
-import { JSDOM } from 'jsdom';
+import unified from 'unified';
+import markdown from 'remark-parse';
+import rehype from 'remark-rehype';
```
Comment (VWSCoronaDashboard3) [inline src/utils/MDToHTMLString.ts:3]:
```suggestion
import remark2rehype from 'remark-rehype';
```

File: src/utils/MDToHTMLString.ts [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+import externalLinks from 'remark-external-links';
 
-import replaceVariablesInText from 'utils/replaceVariablesInText';
+const processor = unified()
+  .use(markdown)
+  .use(externalLinks, {
+    target: false,
+    rel: ['noopener', 'noreferrer'],
+  })
+  .use(rehype)
```
Comment (VWSCoronaDashboard3) [inline src/utils/MDToHTMLString.ts:13]:
```suggestion
  .use(remark2rehype)
```

File: src/pages/over.tsx [hunk 5]
```diff
@@ -23,7 +25,35 @@ interface IVraagEnAntwoord {
...
+interface StaticProps {
+  props: {
+    text: typeof siteText;
+  };
+}
+
+// We use lokalise.com as our dictionary/text source and to support internationalisation.
+// Lokakise will output JSON files which can be found in `src/locale`.
+// However, all content lives inside plain strings. To support structured content and newlines,
+// we (optionally) write markdown in Lokakise and parse it to HTML.
```
Comment (VWSCoronaDashboard3) [inline src/pages/over.tsx:37]: You can put these lines into `MDToHTMLString` and remove the comments from `over` and `verantwoording`. I don’t think it’s needed to duplicate it. You can remove the “Ideally this..” paragraph entirely.

**Review by VWSCoronaDashboard3 — COMMENTED** [review]
Such an improvement 👍

File: src/pages/over.tsx (hunk 2 above)
Comment (VWSCoronaDashboard3) [reply src/pages/over.tsx:42]: AFAIK, this is needed because this function in ran on build-time.

File: src/locale/en.json (hunk 1 above)
Comment (VWSCoronaDashboard3) [reply src/locale/en.json:334]: @VWSCoronaDashboard5 you might have reverted the wrong lokalise PR, it seems. You can revert a revert commit just as you would normally revert a commit.

**Review by VWSCoronaDashboard3 — APPROVED** [review]

**Pull request merged**